releasenotes/notes/feature-datastreams-kombu-bee9eba98bd1f750.yaml

```diff
@@ -0,0 +1,4 @@
+features:
+  - |
+    datastreams: this change adds kombu auto-instrumentation for datastreams monitoring.
+    tracing: this change adds the DD_KOMBU_TRACING_ENABLED flag (default True)
```

Comment:
The correct flag should be `DD_KOMBU_DISTRIBUTED_TRACING`, no?

Reply:
yes, thanks!